09_dynamic_programming/csharp/02_levenshtein/Program.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+
+namespace ConsoleApplication
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var result = LevenshteinDistance("gibraltar", "labrador");
+            Console.WriteLine(result); // 5
+        }
+
+        public static int LevenshteinDistance(string source, string target)
+        {
+            var matrix = CreateMatrix(source, target);
+
+            for (int i = 1; i <= source.Length; i++)
+            {
+                for (int j = 1; j <= target.Length; j++)
+                {
+                    matrix[i, j] = Math.Min(matrix[i, j - 1] + 1, Math.Min(
```

Comment:
this line is a bit hard to read, could you turn it into 2 - 3 lines that are easier to read?